### PR TITLE
fix #554: [PT] Add a script to fetch 'escopos'

### LINKED ISSUE #554: [PT] Add a script to fetch 'escopos'
- Wiktionary page: https://pt.wiktionary.org/wiki/Ajuda:Guia_de_edi%C3%A7%C3%A3o/Lista_de_escopos

This will be used to populate `templates_italic` and fix a lot of issues found by `--check-word`. We just need to find the good page listing of the values.

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #554
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by BoboTiG** [comment]
Do you mind creating a new file, let's say `pt/escopos.py` to allow the automatic script to easily update the list via the gitHub action?

**Comment by lasconic** [comment]
ok; on it.

**Comment by lasconic** [comment]
Ok. It's a bit more involved than expected, so it will take more time. I need to move stuff to last_template_handle and so a lot of tests needs to be move. I will try to have a look tonight.

File: scripts/pt-escopo.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+for line in text.split("\n"):
+    if line.startswith("|"):
+        array = line.split("=")
+        words = array[0].split("|")
+        result = array[1]
+        for word in words:
+            if word and word[0] != "#":
+                results[word] = result
+
+for t, r in sorted(results.items()):
```
Comment (BoboTiG) [inline scripts/pt-escopo.py:31]:
```suggestion
print("escopos = {")
for t, r in sorted(results.items()):
```

**Comment by BoboTiG** [comment]
I was thinking it would be easier, but I may be wrong:

- create `wikidict/lang/pt/escopos.py` (or some other name), it should contain: ```python
"""
List of escopos.
Auto-generated with `python -m scripts`.
"""

# START
# END
```
- in `scripts/__main__.py`, add to `FILES` a line like `"pt-escopos.py": "wikidict/lang/pt/escopos.py",`
- run `python -m scripts`
- In `pt/__init__.py`, do as we've done for FR domain templates: import `escopos` and add them to the `templates_italic`.

Is it a bad approach? I am not sure why you will have to involve `last_template_handler()` 🤔

**Comment by lasconic** [comment]
Ok you are right.

**Comment by lasconic** [comment]
Done.

**Comment by BoboTiG** [comment]
Now that I am looking at #546, should we generate the list with lowercased keys too?

**Comment by lasconic** [comment]
Yes. Just tested with {{escopo|pt|réptil}} --> Zoologia.
Or we do a lower case comparison in lookup_template for pt ?

**Comment by BoboTiG** [comment]
> Or we do a lower case comparison in lookup_template for pt ?

Yes 👍

**Comment by lasconic** [comment]
Done.

**Review by BoboTiG — APPROVED** [review]
Another good thing done 💪

**Pull request merged**